.github/workflows/ci-test.yml

```diff
@@ -51,12 +52,24 @@ jobs:
         with:
           pattern: '^documentation\|^developers.diem.com'
           invert: "true"
+      - id: need-land-blocking-test
+        name: check KS if LBT should be run
+        if: ${{ github.event_name == 'push' }}
+        run: |
+          res=true
+          ks=${{ secrets.KILL_SWITCH_LAND_BLOCKING_TEST }};
+          if ! [[ -z "${ks}" ]] && [[ "${ks}" == ${{ steps.changes.outputs.changes-target-branch }} ]]; then
+            echo "killswitch activated! Will skip land-blocking-test";
+            res=false
+          fi
+          echo "::set-output name=need-lbt::$(echo $res)";
       - id: need-base-images
```

Comment:
I think we can add new switch in this step as well which can avoid a case we do a base image build for around 20mins but we actually do not need to run LBT

Reply:
only case the step returns True should be both flags set to true